### ISSUE TITLE
Add virtual list support

### ISSUE THREAD

**Issue by pam-param-pam** [filebrowser/filebrowser#3598]
Right now, filebrowser gets very slow when a folder has >200 items in it.

One possible sollution is to make use of virtual lists. For example this library:
https://github.com/Akryum/vue-virtual-scroller/tree/master

Sadly im not experienced enough to make PR, i tried and got weird css/hover issues.

**Comment by gtsteffaniak** [comment]
My fork has UX imporvements I think you will find works better : https://github.com/gtsteffaniak/filebrowser

**Comment by pam-param-pam** [comment]
@gtsteffaniak Hi, thanks for ur reply. Your UX improvements look great. Hovewer after inspecting source code I noticed it does not use virtuall lists, so sadly it doesn't solve the problem of displaying hundreds of items at the same time

**Comment by gtsteffaniak** [comment]
Yes it doesn't use virtual lists, but it can efficiently load 10k items at a time. 

Filebrowser/filebrowser uses lazy loading API in the UI which batches loading only what's on screen. The branch I have load everything from the server efficiently, quickly handling up to 10k items on a standard 4G connection.

If your internet connection is faster, it could handle 100k items within a few seconds.

Sorting and changing view in UI is instant and does not require reloading the data from the backend.
